rootfs/templates/wrapper-body.sh

```
@@ -0,0 +1,589 @@
+
+## Read in custom configuration here, so it can override defaults
+
+export GEODESIC_CONFIG_HOME="${XDG_CONFIG_HOME:-$HOME/.config}/geodesic"
+if ! [ -d "$GEODESIC_CONFIG_HOME" ] && [ -d "$HOME/.geodesic" ]; then
+	GEODESIC_CONFIG_HOME="$HOME/.geodesic"
+fi
+
+verbose_buffer=()
+launch_options="$GEODESIC_CONFIG_HOME/defaults/launch-options.sh"
+if [ -f "$launch_options" ]; then
+	source "$launch_options" && verbose_buffer+=("Configuration loaded from $launch_options") || printf 'Error loading configuration from %s\n' "$launch_options" >&2
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)